okcurl/build.gradle.kts

```diff
@@ -36,14 +37,19 @@ tasks.getByName<Jar>("sourcesJar") {
   dependsOn("copyResourcesTemplates")
 }
 
+kapt {
+  arguments {
+    arg("project", "${project.group}/${project.name}")
```

Comment:
Is this really needed, or just an example for configuration in the docs?

Reply:
Actually I think it's not necessary.